src/privileged_extensions.c

```diff
@@ -157,7 +158,7 @@ void handle_drop_extension(void (*process_utility_hook)(PROCESS_UTILITY_PARAMS),
     ListCell *lc;
 
     foreach (lc, stmt->objects) {
-        char *name = strVal((Value *)lfirst(lc));
+        char *name = strVal(lfirst(lc));
```

Comment:
`Value` is not exposed on pg 15. I didn't see any issue when removing it(tests pass)